files/image_config/process-reboot-cause/process-reboot-cause

```diff
@@ -82,6 +82,8 @@ def find_software_reboot_cause():
         log_info("Reboot cause file {} not found".format(REBOOT_CAUSE_FILE))
 
     if os.path.isfile(FIRST_BOOT_PLATFORM_FILE):
+        if software_reboot_cause == UNKNOWN_REBOOT_CAUSE:
+            software_reboot_cause = "SONiC image installation"
```

Comment:
should we put all reboot reason to one place as macro and then use it, then we would know what reboot reasons we can have.

Reply:
Done. Also renamed `UNKNOWN_REBOOT_CAUSE` to `REBOOT_CAUSE_UNKNOWN` so that constant names have consistent prefixes.